### ISSUE TITLE
This!

### ISSUE THREAD

**Issue by sl1pm4t** [ironbay/snooze#1]
This is so awesome it should be illegal.
Thanks @dax0 !

**Comment by janajri** [comment]
I agree, I use it for everything and anything. Even found a way to embed this tiny package into my toaster so that it can handle api calls to ready my toast as soon as my alarm goes off.
